pkg/jx/cmd/opts/import.go

```diff
@@ -211,7 +211,6 @@ func (o *CommonOptions) LogImportedProject(isEnvironment bool, gitInfo *gits.Git
 	if !isEnvironment {
 		log.Infof("Watch pipeline activity via:    %s\n", util.ColorInfo(fmt.Sprintf("jx get activity -f %s -w", gitInfo.Name)))
 		log.Infof("Browse the pipeline log via:    %s\n", util.ColorInfo(fmt.Sprintf("jx get build logs %s", gitInfo.PipelinePath())))
-		log.Infof("Open the Jenkins console via    %s\n", util.ColorInfo("jx console"))
```

Comment:
So we remove this message regardless of the install type? Works for me. 
 I wonder what should happen long term with `jx console` altogether. I think it should be removed.